src/Nox.Types/Types/CultureCode/CultureCode.cs

```diff
@@ -7,6 +14,13 @@ namespace Nox.Types;
 /// </summary>
 public partial class CultureCode : ValueObject<string, CultureCode>
 {
+    private static readonly ImmutableHashSet<string> SupportedCultureCodes = System.Enum.GetValues(typeof(Culture)).Cast<Culture>().Select(v =>
```

Comment:
this is being computed in other place, we could make this an extension method to CultureCode... eventually keep it in a static field, so we have only a single instance in memory

Reply:
Desired changens have been applied